libkpod/config.go

```diff
@@ -0,0 +1,38 @@
+package libkpod
+
+// Config stores the configuration options for libkpod
+type Config struct {
+	Root                     string
+	RunRoot                  string
+	Storage                  string
+	StorageOptions           []string
+	LogDir                   string
+	Runtime                  string
+	RuntimeUntrustedWorkload string
+	DefaultWorkloadTrust     string
+	Conmon                   string
+	ConmonEnv                []string
+	CgroupManager            string
+	DefaultTransport         string
+	InsecureRegistries       []string
+	SignaturePolicyPath      string
+}
+
+// DefaultConfig returns a Config struct with all options set to the default
+func DefaultConfig() *Config {
```

Comment:
At this point, I think it would be better to just remove this, and trust kpod, buildah, and CRI-O to handle configuration and defaults themselves and assume they pass us a sane config. We can consider how best to integrate the c/storage and c/image default configs in a later PR.